src/pkg/reg/adapter/aliacr/adapter.go

```diff
@@ -32,7 +34,16 @@ import (
 	"github.com/goharbor/harbor/src/pkg/registry/auth/bearer"
 )
 
+var (
+	acrQPSLimit = 20
+)
+
 func init() {
+	var envAcrQPSLimit, _ = strconv.Atoi(os.Getenv("ACR_QPS_LIMIT"))
+	if envAcrQPSLimit > 1 && envAcrQPSLimit < acrQPSLimit {
```

Comment:
This means that the QPS specified from ENV can only less than the fixed `acrQPSLimit` defined above?

Reply:
yes